integrations/test-gitlab-v2/gitlab/api.py

```diff
@@ -0,0 +1,15 @@
+import os
+import aiohttp
+from typing import Any, List
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
Environment variables are loaded by default when you define them in the `spec.yaml` file. Check other integrations such as [Datadog](https://github.com/port-labs/ocean/blob/main/integrations/datadog/main.py#L23) to see how this is implemented